tstest/integration/vms/vms_test.go

```diff
@@ -242,7 +242,7 @@ func fetchFromS3(t *testing.T, fout *os.File, d Distro) bool {
 
 // fetchDistro fetches a distribution from the internet if it doesn't already exist locally. It
 // also validates the sha256 sum from a known good hash.
-func fetchDistro(t *testing.T, resultDistro Distro, bins *integration.Binaries) string {
+func (h Harness) fetchDistro(t *testing.T, resultDistro Distro) string {
```

Comment:
Should these be pointer-recievers, i.e. *Harness?

Reply:
They aren't mutating the values of the Harness struct and this is non-hot code. I'd prefer to take the "immutability" over speedhacking a test with a pointer receiver.